internal/api/v1/handlers/instance.go

```diff
@@ -46,7 +46,7 @@ func (h *InstanceHandler) ListInstances(c *fiber.Ctx) error {
 
 	// TODO: should check for OwnerID and filter by it
 
-	instances, err := h.service.ListInstances(c.Context(), &opts)
+	instances, err := h.service.ListInstances(c.Context(), models.AdminID, &opts)
```

Comment:
is `models.AdminID` a predefined constant to be stored in DB?
If so this would list only the instances that are created by that particular AdminID if we do not have a check in services for it. 
Also I think going role based gives us a better flexibility in managing items and users.

Reply:
https://github.com/celestiaorg/talis/issues/151